package-lock.json

```diff
@@ -1,8 +1,9928 @@
 {
   "name": "noobaa-core",
   "version": "5.12.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
By this line, it was done using npm > 6 (7 or 8)
please revert this and use npm v6 for this (and avoid possible brakes in the DS)
We should bump npm from v6 to v8 as part of bumping node 14 to 16

Reply:
Ah that's right, completely missed that. It was npmv8. Will revert this and will re-generate a new lock file.